### ISSUE TITLE
Allow to work without plugins

### ISSUE THREAD

**Issue by ai** [postcss/postcss-cli#33]
`postcss-cli` could be very useful to convert [SugarSS](https://github.com/postcss/sugarss) to CSS or CSS to SugarSS.

But I got error:

```
postcss -p sugarss test.sss -o test.css
Usage: /home/ai/test/node_modules/.bin/postcss [--use|-u] plugin [--config|-c
config.json] [--output|-o output.css] [input.css]

…

Please specify at least one plugin name.
```

**Comment by ai** [comment]
@watilde we could miss plugins count check if user set parser or stringifier

**Comment by ai** [comment]
@watilde any thoughts? Do you need PR? Say something ;)

**Comment by RyanZim** [comment]
@watilde bump, bump

**Comment by watilde** [comment]
I apologize for my late late response to all! I totally agreed with this change so I will create a patch.

**Comment by watilde** [comment]
This is fixed by `v2.6.0`! Please try it with new version: `npm i -g postcss-cli`.
